apps/v3_api/lib/mock_routes.ex

```diff
@@ -0,0 +1,1762 @@
+defmodule V3Api.MockRoutes do
+  @moduledoc """
+  A mock Routes API client for testing purposes.
+  """
+
+  @behaviour V3Api.RoutesApi
+
+  alias JsonApi.Item
+
+  @red %Item{
```

Comment:
Is there a better way of doing this than making huge modules with module attributes like this?

Reply:
That's partly why I suggested doing it at the `Routes.Repo` level. There, you'd only need to provide the data that's a part of the `%Route{}` struct instead of a full JSON:API response.